src/biblenotify/core/notifications.py

```diff
@@ -96,7 +98,43 @@ def loadChapter(self, location: str) -> dict:
 
         contents_json = json.loads(contents_string)
```

Comment:
Notice that this project uses camelCase rather than Python's standard snake_case. This is because of Qt. For future knowledge, generally you adapt to whatever standard the project code-base already uses. In this case, we should refactor the rest of the file to camelCase. I and @iwoithe must have let this slip by. 😄 

In this review I will use the camelCase variable naming even when referring to varables currently snake_case.

Reply:
Oh ok